.changeset/engine-catch-outside-atomic.md

```
@@ -0,0 +1,55 @@
+---
+'@substrat-run/boundary-lint': minor
+---
+
+Catching an engine error outside `ctx.atomic` is now a lint error
+
+#770 landed `ctx.atomic`, so a vertical *can* catch an engine error safely. It only moved
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)